packages/app/src/systems/Core/utils/provider.ts

```diff
@@ -0,0 +1,5 @@
+import { Provider } from 'fuels';
+
+import { VITE_FUEL_PROVIDER_URL } from '~/config';
+
+export const provider = new Provider(VITE_FUEL_PROVIDER_URL);
```

Comment:
This should be on __mocks__ to avoid being used on other places of the applications, as the provider can be changed on the network selector.